packages/arb-token-bridge-ui/src/util/TokenListUtils.ts

```diff
@@ -127,8 +127,6 @@ export interface TokenListWithId extends TokenList {
 export const validateTokenList = (tokenList: TokenList) => {
   const ajv = new Ajv()
   addFormats(ajv)
-  // https://github.com/OffchainLabs/arbitrum-token-lists/blob/master/src/lib/validateTokenList.ts#L10
-  schema.properties.tokens.maxItems = 15_000
```

Comment:
Now that we remove irrelevant tokens from token list, the biggest list is around ~6k tokens. I also removed this change in token list generation, so any list with over 10k items would fail generation.